Module/src/NewWordCloudCommand.cs

```diff
@@ -973,7 +961,29 @@ private static IEnumerable<string> SortWordList(IDictionary<string, float> dicti
             } while (clockwise ? angle <= maxAngle : angle >= maxAngle);
         }
 
-        private static float RandomFloat() => (float)Random.NextDouble();
+        private static float RandomFloat()
+        {
+            lock (_randomLock)
+            {
+                return (float)Random.NextDouble();
+            }
+        }
+
+        private static int RandomInt()
+        {
+            lock (_randomLock)
+            {
+                return Random.Next();
+            }
+        }
+
+        private static void Shuffle<T>(T[] items)
+        {
+            lock (_randomLock)
+            {
+                Random.Shuffle(items)
```

Comment:
Does this resolve? :)

Reply:
Surprisingly, yes! It seems to infer the generic type argument from the parameter type, since they both refer to the same type.

Reply:
nvm, I thought you'd removed the extension method, I'll show myself out 😄